app/assets/stylesheets/common/components/user-status-picker.scss

```diff
@@ -0,0 +1,35 @@
+.user-status-picker-wrap {
+  display: inline-flex;
+  width: 100%;
+  height: 2.3em;
```

Comment:
Does height need to be set here? Seems redundant.

Reply:
This was redundant, I've removed it.